GetFailedUserDataProcessing/handler.ts

```diff
@@ -53,43 +54,40 @@ export const GetFailedUserDataProcessingHandler = (
   | IResponseErrorInternal
   | IResponseErrorNotFound
 > =>
-  tryCatch(
-    () =>
-      new Promise<Option<TableEntry>>((resolve, reject) =>
-        tableService.retrieveEntity(
-          failedUserDataProcessingTable,
-          choice,
-          fiscalCode,
-          null,
-          (error: Error, result: TableEntry, response: ServiceResponse) =>
-            response.isSuccessful
-              ? resolve(some(result))
-              : response.statusCode === 404
-              ? resolve(none)
-              : reject(error)
-        )
-      ),
-    toError
-  )
-    .mapLeft<IResponseErrorInternal | IResponseErrorNotFound>(er =>
-      ResponseErrorInternal(er.message)
-    )
-    .chain(maybeTableEntry =>
-      fromEither(
-        fromOption(ResponseErrorNotFound("Not found!", "No record found."))(
-          maybeTableEntry
-        )
+  pipe(
+    TE.tryCatch(
+      () =>
+        new Promise<O.Option<TableEntry>>((resolve, reject) =>
+          tableService.retrieveEntity(
+            failedUserDataProcessingTable,
+            choice,
+            fiscalCode,
+            null,
+            (error: Error, result: TableEntry, response: ServiceResponse) =>
+              response.isSuccessful
+                ? resolve(O.some(result))
+                : response.statusCode === 404
+                ? resolve(O.none)
+                : reject(error)
+          )
+        ),
+      E.toError
+    ),
+    TE.mapLeft(er => ResponseErrorInternal(er.message)),
+    TE.chainW(
+      flow(
+        E.fromOption(() =>
+          ResponseErrorNotFound("Not found!", "No record found.")
+        ),
+        E.map(rs => ({
+          failedDataProcessingUser: rs.RowKey._
+        })),
+        TE.fromEither
```

Comment:
```suggestion
        TE.fromOption(() =>
          ResponseErrorNotFound("Not found!", "No record found.")
        ),
        TE.map(rs => ({
          failedDataProcessingUser: rs.RowKey._
        }))
```

Reply:
fixed in https://github.com/pagopa/io-functions-admin/pull/165/commits/1c8838d02fa5bf56455a7db15d0e1d9ec72b7aec